### DISCUSSION TITLE
Jump to qcxf 3.8.0

### DISCUSSION THREAD

**Discussion by famod** [quarkiverse/quarkus-cxf#1245] [General]
Hi @ppalaga,

what's the rationale for jumping from 2.7.1 to 3.8.0? Is the Quarkus version now leading the way for the qcxf version or was it a mistake and should have been 2.8.0 instead?

Disclaimer: I was about to create the GH release for 3.8.0.

Cheers!

**Comment by famod** [comment]
Ok, I've just found https://github.com/quarkiverse/quarkus-cxf/pull/1240/files#diff-05fc01ddb0c378e4108dff91972479ccfb51894066ed11c13de8d115ce6550b8

How do we want to proceed with GH releases? Simply add a link to https://docs.quarkiverse.io/quarkus-cxf/dev/release-notes/index.html or more specifically to https://docs.quarkiverse.io/quarkus-cxf/dev/release-notes/3.8.0.html (in this case)?

**Reply by ppalaga** [reply]
Yeah, I went with 3.8.0 to be aligned with Quarkus. 

> How do we want to proceed with GH releases? Simply add a link to https://docs.quarkiverse.io/quarkus-cxf/dev/release-notes/index.html or more specifically to https://docs.quarkiverse.io/quarkus-cxf/dev/release-notes/3.8.0.html (in this case)?

Yeah, that sounds good, as long as the release notes for the given version exists. I hope I'll be able to keep writing those in the future.

**Reply by famod** [reply]
Hey @ppalaga, I was about to create the 3.10.0 release entry but the docs don't have 3.10.0 release notes yet. (When) are you planning to add those?
Thanks!

**Reply by ppalaga** [reply]
Mañana :) /me just got back from PTO

**Reply by ppalaga** [reply]
3.10.0 should appear on https://docs.quarkiverse.io/quarkus-cxf/dev/release-notes/index.html soon.

**Reply by famod** [reply]
So I noticed that quarkiverse-ci now auto-creates GH releases, which is nice - e.g.: https://github.com/quarkiverse/quarkus-cxf/releases/tag/3.16.0

But that also means there is no link to the release notes in the docs anymore, unless someone adds it manually. :man_shrugging:

**Reply by ppalaga** [reply]
Yes, indeed. @gastaldi can you perhaps see any way how we could smuggle our release notes URL into the GH release description? 
Our release notes URLs always follow the same pattern: `https://docs.quarkiverse.io/quarkus-cxf/dev/release-notes/${version}.html`

**Reply by gastaldi** [reply]
One solution is to pass `create_release: false` to the prepare-release.yml file to skip the release creation there and do it in your local `.github/workflow/prepare-release.yml` when the job finishes (https://github.com/quarkiverse/.github/commit/90ac21d1d150a0fa6e698a5a86baebbcb93e4202)

**Reply by gastaldi** [reply]
Or a simpler solution is to edit the release and set the Release Notes as needed. Here is a PR doing this: https://github.com/quarkiverse/quarkus-cxf/pull/1576
